web/src/app/browser/src/utilApiEndpoint.ts

```diff
@@ -23,6 +23,10 @@ export function createUnselectableElement<E extends keyof HTMLElementTagNameMap>
   return e;
 }
 
+/**
+ * Defines the base object for the long-standing `keyman.util` API methods, maintaining
+ * their long-standing names and signatures.
+ */
```

Comment:
Uh... that's because it's mostly a do-nothing class.  It only exists to ensure that `keyman.util` exists with its public API endpoints as defined on help.keyman.com.

```suggestion
/**
 * Defines the base object for the long-standing `keyman.util` API methods, maintaining
 * their long-standing names and signatures as defined at
 * https://help.keyman.com/developer/engine/web/current-version/reference/util/
 */
```

Reply:
```suggestion
/**
 * Defines the base object for the long-standing `keyman.util` API methods, maintaining
 * their published names and signatures as defined at
 * https://help.keyman.com/developer/engine/web/current-version/reference/util/
 */
```